src/components/fields/image-upload/image-input/image-input.styles.ts

```diff
@@ -12,7 +11,7 @@ export interface SubtitleProps {
 export const Wrapper = styled.div`
 	border-radius: 0.25rem;
 	${(props) => {
-		const color = encodeURIComponent(V2_Color.Neutral[5](props));
+		const color = encodeURIComponent(Colour["bg-strongest"](props));
```

Comment:
as above, suggest to replace this entire part with Border.Util["dashed-default"] from the [Border](https://designsystem.life.gov.sg/react/index.html?path=/docs/foundations-border-introduction--docs) tokens